commons/src/main/scala/dagr/commons/reflect/ReflectiveBuilder.scala

```diff
@@ -92,6 +114,7 @@ class ReflectiveBuilder[T](val clazz: Class[T]) {
             case (Some(mem), Some(mir)) =>
               val defarg = mem.member(TermName("$lessinit$greater$default$" + (index + 1)))
               if (defarg != NoSymbol) Option(mir.reflectMethod(defarg.asMethod)()) else None
+            case _ => unreachable("Should not reach here.")
```

Comment:
Can you make the message more specific, so that if it ever does happen it's clearer?  E.g. `unreachable("companionMembers and instanceMirror must both be one or Some")`